auditapi/api_events.go

```diff
@@ -13,17 +13,13 @@ package auditapi
 
 import (
 	"bytes"
-	_context "context"
```

Comment:
Please make sure that the template we are using is from the proper version of openapi-generator (5.2.x)

Reply:
Changed template